packages/angular_devkit/build_optimizer/BUILD.bazel

```diff
@@ -4,13 +4,8 @@
 # found in the LICENSE file at https://angular.io/license
 
 load("@npm//@bazel/jasmine:index.bzl", "jasmine_node_test")
-load("//tools:defaults.bzl", "ts_library")
-
-# @external_begin
-load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
-load("@build_bazel_rules_nodejs//:index.bzl", "pkg_npm")
+load("//tools:defaults.bzl", "pkg_npm", "ts_library")
 load("@npm//@angular/dev-infra-private/bazel/api-golden:index.bzl", "api_golden_test_npm_package")
```

Comment:
Keep `@external_begin & end` around the load from @npm. That doesn't exist in google3.

Reply:
Nevermind. Not needed as per https://github.com/angular/angular-cli/pull/22258#discussion_r759582051